BRAIN/IsPedActiveInScenario.md

```diff
@@ -8,8 +8,11 @@ ns: BRAIN
 BOOL IS_PED_ACTIVE_IN_SCENARIO(Ped ped);
 ```
 
+```
```

Comment:
Same as above, please remove the code block from the description if it's not actually any code.